docs/user-guide/relay-nodes.md

```diff
@@ -2,13 +2,30 @@
 
 The Nexodus Service makes the best effort to establish direct peering between devices, but in some scenarios such as symmetric NAT, it's not possible to establish direct peering. To establish connectivity in those scenarios, the Nexodus Service uses a relay node to relay the traffic between the endpoints.
 
-The Nexodus Service does not offer public, shared relay nodes. Instead, a relay node must be added to each VPC which requires them. Adding a relay node follows the same process as any other device but with additional options given to `nexd`.
+Currently Nexodus supports two types of relay:
 
-A relay node needs to be reachable on a predictable Wireguard port such as the default UDP port of 51820. They would most commonly be run on a public IP address, though it could be anywhere reachable by all devices in the VPC. There is only a need for one relay node in a VPC.
+1. Wireguard based relay :
+    This relay node does not offer public, shared relay nodes. Instead, a relay node must be added to each VPC which requires them. Adding a relay node follows the same process as any other device but with additional options given to `nexd`. This relays traffic using the wireguard tunnels, so the traffic gets decrypted and re-encrypted at the relay node. This relay node is hosted by the customer and can be any device that can run the Nexodus Agent.
+2. HTTPS based relay (DERP Relay) :
+    This relay node is a public, shared relay node that is hosted by Nexodus at [Nexodus DERP Relay](relay.nexodus.io). This relay node relays traffic using HTTPS/TLS, so the traffic is not decrypted at the relay node. This relay node is hosted by Nexodus and is available to all users. Scenarios where user don't want to use the public relay, user can manually deploy this relay on their own infrastructure and onboard the relay similar to the wireguard relay node, but as a wireguard relay, it would be scoped to the specific VPC where it's on-boarded.
+
+Given that both the relays can be on-boarded manually and relays the traffic, the reason we support wireguard based relay is because it performs relatively better compared to HTTPS/TLS relay.
+
+A relay node needs to be reachable from all the devices to ensure that devices can successfully connect (wireguard/udp, https/tcp) to the relay (and also on a predictable Wireguard port such as the default UDP port of 51820 for wireguard based relay). They would most commonly be run on a public IP address, though it could be anywhere reachable by all devices in the VPC. There is only a need for one relay node in a VPC.
```

Comment:
```suggestion
A relay node needs to be reachable from all the devices to ensure that devices can successfully connect (wireguard/udp, https/tcp) to the relay (and also on a predictable Wireguard port such as the default UDP port of 51820 for wireguard-based relay). They would most commonly be run on a public IP address, though it could be anywhere reachable by all devices in the VPC. There is only a need for one relay node in a VPC.
```